src/components/RegisterLoginModal/RegisterLoginModal.jsx

```diff
@@ -43,9 +45,16 @@ function RegisterLoginModal() {
             bgcolor: 'background.paper',
           }}
         >
-          <RegisterLoginHeader checked={checked} onChange={handleChange} />
-          {checked ? <RegisterForm fn={changeState} /> : <LoginForm fn={changeState} />}
-          {state ? <ActionAlert fn={setState} children={state} /> : null}
+          <IconButton aria-label="Close" className={styles.closeButton} onClick={handleClose}>
+            <CloseIcon />
+          </IconButton>
+          <RegisterLoginHeader checked={checked} onChange={handleChange} onClick={handleClose} state={state} />
+          {checked ? <RegisterForm changeStateFn={changeState} /> : <LoginForm changeStateFn={changeState} />}
+          {state === 'Success' ? (
+            <ActionAlert severity="success" onCloseFn={setState} children={state} />
```

Comment:
Again, I encourage to not use children as `props` and pass them like that instead:
```
            <ActionAlert severity="success" onCloseFn={setState}>
              {state}
            </ActionAlert>
```
The rule `"react/no-children-prop": "off"` won't be needed then.

Reply:
Unfortunately, we I make it like this the description doesn't appear. I think it's because internal rules of MUI 
https://mui.com/api/alert/
![obraz](https://user-images.githubusercontent.com/80857044/152884495-36b50156-469e-4a0d-8fde-c8bbd2a00df6.png)

Reply:
Ah, I see.